client/my-sites/sidebar/index.jsx

```diff
@@ -678,35 +678,22 @@ export class MySitesSidebar extends Component {
 	};
 
 	store() {
-		const {
-			translate,
-			site,
-			siteSuffix,
-			canUserUseWooCommerceCoreStore,
-			isSiteWpcomStore,
-		} = this.props;
+		const { translate, site, canUserUseWooCommerceCoreStore, isSiteWpcomStore } = this.props;
 
 		if ( ! site ) {
 			return null;
 		}
 
-		let experience = 'calypso-store';
-		let storeLink = '/store' + siteSuffix;
-		if ( isEcommerce( site.plan ) && canUserUseWooCommerceCoreStore ) {
-			// Eventually, the plan is to have the WooCommerce Core menu item labelled the same
-			// for both Business and eCommerce users. But, for now, we want to continue to
-			// use the "Store" label for eCommerce users because that is what they are used to.
-			// So, we'll just continue to change the link here as we have been doing.
-			experience = 'wpadmin-woocommerce-core';
-			storeLink = site.options.admin_url + 'admin.php?page=wc-admin';
-		} else {
+		if ( ! isEcommerce( site.plan ) || ! canUserUseWooCommerceCoreStore ) {
 			return null;
 		}
 
 		if ( ! isSiteWpcomStore && isBusiness( site.plan ) ) {
 			return null;
 		}
 
+		const storeLink = site.options.admin_url + 'admin.php?page=wc-admin';
```

Comment:
In theory this can be fragile, but it was like that before, so let's leave it as-is.

Reply:
How can it be fragile? `site.options` may not exist? Or is the constructed URL sometimes wrong?

Reply:
As an option, it can be altered in the database, or get filtered on retrieval by plugins. That might change its format in unexpected ways, add / remove trailingslashes, add additional query arguments, etc.